src/bioregistry/analysis/paper_ranking.py

```diff
@@ -397,7 +397,9 @@ def main(bioregistry_file: Path, start_date: str, end_date: str) -> None:
     importances_df.to_csv(importance_path, sep="\t", index=False)
 
     # These have already been curated and will therefore be filtered out
-    curated_pmids = set(curated_papers_df["pubmed"])
+    curated_pmids = set(curated_papers_df["pubmed"]).union(
+    publication_df["pubmed"], curation_df["pubmed"]
```

Comment:
can you look into why CI isn't picking up on this formatting error?

Reply:
I'm not entirely sure. I can't find anything in the logs that would explain this. I did recently switch to using a new laptop recently which has been causing a wide range of issues but I don't think anything in my local environment should affect the CI from picking up formatting errors?

Reply:
I found this surprising as well, don't see any obvious explanation. In any case, the indentation fixed as far as this PR goes.